crates/node-maintainer/Cargo.toml

```diff
@@ -11,6 +11,7 @@ oro-package-spec = { path = "../oro-package-spec" }
 
 async-std = { workspace = true }
 futures = { workspace = true }
+indicatif = { workspace = true }
```

Comment:
This should only be included for `not(target_arch = "wasm32")`, since `indicatif` won't be usable at all in wasm contexts, and node-maintainer is meant to be used there.